wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/environment/server/DefaultServicesStartStrategy.java

```diff
@@ -132,7 +156,22 @@ private String getServiceFromMachineLink(String link) throws IllegalArgumentExce
         if (link != null) {
             String[] split = service.split(":");
             if (split.length != 1 && split.length != 2) {
-                throw new IllegalArgumentException(format("Service link %s is invalid", link));
+                throw new IllegalArgumentException(format("Service link '%s' is invalid", link));
+            }
+            service = split[0];
+        }
+        return service;
+    }
+
+    /**
+     * Parses volumesFrom content into depends_on field representation - removes column and further chars
+     */
+    private String getServiceFromVolumesFrom(String volumesFrom) throws IllegalArgumentException {
```

Comment:
The same code is in `getServiceFromMachineLink` method, are you sure it can't be reused?